src/main/webapp/download.html

```diff
@@ -67,19 +67,19 @@ <h2 class="pb-3 mb-2"><i class="fas fa-download"></i> Download</h2>
           <div class="card flex-md-row mb-4 box-shadow h-md-250">
             <div class="card-body d-flex flex-column align-items-start">
               <strong class="d-inline-block mb-2 text-success"><i class="fas fa-circle"></i> Latest release</strong>
-              <h3 class="mb-0 text-dark">Karaf Runtime <span class="text-muted">4.2.2</span></h3>
-              <div class="mb-1 text-muted">December 18, 2018</div>
+              <h3 class="mb-0 text-dark">Karaf Runtime <span class="text-muted">4.2.3</span></h3>
+              <div class="mb-1 text-muted">February 7, 2019</div>
```

Comment:
The date is actually February, 11, 2019.

Reply:
done